llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -3316,8 +3328,10 @@ cc_library(
     ]),
 )
 
-cc_binary(
-    name = "llvm-rc",
+# Simulate the upstream alias by just creating the target with two different
```

Comment:
Aw man I created a whole starlark rule for doing exactly this that's used elsewhere. Let's not do this here. I sent a change to google internal to use `binary_alias` instead. Of course GitHub is being stupid and won't let me leave a suggested edit that crosses lines it considers unchanged :roll_eyes: , so I've pushed a commit that makes the change I'd like to see